build.sh

```diff
@@ -1 +1,6 @@
-python gen-PKGBUILD.py > PKGBUILD
+if [[ `pacman -Q python-debian 2>&1` == "error: package 'python-debian' was not found" ]]; then
```

Comment:
Wouldn't it be better / safer to catch the ImportError in gen-PKGBUILD.py and print the message there?
Someone may call it directly... I always did. ^^

Reply:
Good idea. It will also make it more universal in a way that you can use it in debian.
But still there is a problem for debian because of using makepkg in a git hook. It is used for regenerating .srcinfo. Maybe we could somehow workaround it too.